homeassistant/components/guardian/strings.json

```
@@ -6,17 +6,17 @@
         "description": "Configure a local Elexa Guardian device.",
         "data": {
           "ip_address": "[%key:common::config_flow::data::ip%]",
-          "port": "Port"
+          "port": "[%key:common::config_flow::data::port%]"
         }
       },
       "zeroconf_confirm": {
         "description": "Do you want to set up this Guardian device?"
       }
     },
     "abort": {
-      "already_configured": "This Guardian device has already been configured.",
+      "already_configured": "[%key:common::config_flow::abort::already_configured_device%]",
       "already_in_progress": "Guardian device configuration is already in process.",
-      "connection_error": "Failed to connect to the Guardian device."
+      "connection_error": "[%key:common::config_flow::error::cannot_connect%]"
```

Oh my bad, one more thing : 
```suggestion
      "cannot_connect": "[%key:common::config_flow::error::cannot_connect%]"
```
And update `config_flow.py` and tests to reflect this change